codemeta.json

```diff
@@ -0,0 +1,52 @@
+{
+  "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+  "@type": "Code",
+  "name": "Berkeley Database Information Index (BDII)",
+  "description": "The BDII implementation of the information",
+  "provider": {
+    "@type": "Organization",
+    "name": "",
+    "url": ""
+  },
+  "maintainer": [
+    {
+      "@type": "Person",
+      "@id": "https://orcid.org/0000-0002-5686-3193",
+      "name": " Baptiste",
+      "familyName": "Grenier",
+      "affiliation": {
+        "@type": "Organization",
+        "name": "EGI Foundation",
+        "url": "http://www.egi.eu"
+      }
+    },
+    {
+      "@type": "Role",
+      "roleName": "Security and Vulnerability",
+      "url": ""
+    }
+  ],
+  "operatingSystem": [
+    "LINUX-2.4",
+    "centos 6",
+    "centos 7"
+  ],
+  "installUrl": "https://malandes.web.cern.ch/malandes/infosys/rpms/emi-bdii-site-1.0.2-1.el7.cern.noarch.rpm",
+  "buildInstructions": "http://gridinfo-documentation.readthedocs.io/",
+  "releaseNotes": "",
+  "codeRepository": "https://github.com/EGI-Foundation/bdii",
+  "contIntegration": "https://travis-ci.org/EGI-Foundation/bdii",
+  "networkStack": "",
+  "supportUnit": {
+    "@type": "contactPoint",
+    "contactType": "GGUS",
+    "name": "Information System Development"
+  },
+  "developmentStatus": "supported",
+  "author": [],
+  "citation": "",
+  "dateCreated": "",
+  "dateModified": "",
+  "keywords": [],
+  "license": ""
```

Comment:
Apache 2.0 for the License, for the author can we point to a file like https://github.com/EGI-Foundation/bdii/blob/master/AUTHORS ?
Please also add a new line at EOF.

Reply:
We can't point to a file, because that would break the schema. But we can write the authors based off of AUTHORS.md

AUTHORS and LICENSE are actually made redundant by this metadata - but some systems and tools do not read it, preferring separate files. So, I suggest we adopt a convention of contributors sending pull requests to AUTHORS.md and CONTRIBUTORS.md and maintainers updating `codemeta.json`